cclib/bridge/cclib2horton.py

```diff
@@ -82,7 +82,40 @@ def makehorton(ccdat):
             attributes["npa_charges"] = ccdat.atomcharges["natural"]
 
     elif hortonver == 3:
-        pass  # Populate with bridge for horton 3 in later PR
+        # In horton 3 IOData, inputs are type-checked -- thus the bridge also verifies the types.
+        # if coordinates are known; numpy.ndarray (size natom-by-3)
+        if hasattr(ccdat, "atomcoords") and isinstance(ccdat.atomcoords, numpy.ndarray):
```

Comment:
Let's use `np.asanyarray` to do conversions throughout again, otherwise these attributes will silently get ignored when you really want them to be added.

Though, at the point you have a `ccData` instance, all of these should already be `ndarray`s. so I wonder if any of that is necessary.

Reply:
I've removed the unnecessary checks you have mentioned. Thanks for the comment!